jobs/opensearch_templates/spec

```diff
@@ -21,12 +23,18 @@ properties:
     description: |
       Name prefix of your `app` log indices. If you don't split `app` and `platform` indices, then just set it with the value of `opensearch_config.index_prefix`.
     default: "logs-app"
+  opensearch_config.metric_index_prefix:
+    description: |
+      Name prefix of your `app` log indices. If you don't split `app` and `platform` indices, then just set it with the value of `opensearch_config.index_prefix`.
+    default: "logs-metrics"
```

Comment:
If the indices are named `log-metrics*`, won't it require a new index pattern to search them? Or is that the idea? So that metrics don't appear by default when you search on the Discover page

Reply:
correct, these are a seperate index